packages/next/build/webpack/plugins/webpack-conformance-plugin/checks/minification-conformance-check.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  IWebpackConformanceTest,
+  IConformanceTestResult,
+} from '../TestInterface'
+import { CONFORMANCE_ERROR_PREFIX } from '../constants'
+
+export class MinificationConformanceCheck implements IWebpackConformanceTest {
+  public buildStared(options: any): IConformanceTestResult {
+    // TODO(prateekbh@): Implement warning for using Terser maybe?
+
+    if (options.optimization.minimize === false) {
+      return {
+        result: 'FAILED',
```

Comment:
Probably better to use a TypeScript enum for this.

Reply:
done